src/main/java/com/denizenscript/depenizen/bukkit/properties/essentials/EssentialsPlayerProperties.java

```diff
@@ -301,6 +301,22 @@ public void adjust(Mechanism mechanism) {
             }
         }
 
+        // <--[mechanism]
+        // @object PlayerTag
+        // @name remove_essentials_home
+        // @input ElementTag(String)
+        // @plugin Depenizen, Essentials
+        // @description
+        // Removes the specified player's Essentials home.
+        // -->
+        if (mechanism.matches("remove_essentials_home")) {
+            try {
+                getUser().delHome(mechanism.getValue().toString());
+            } catch (Exception e) {
+                Debug.echoError(e);
```

Comment:
When/why do exceptions happen? Does it happen if the name is invalid? If so, that should be better handled.